azext_iot/central/commands_device.py

```diff
@@ -68,10 +68,16 @@ def registration_info(
     device_id=None,
     token=None,
     central_dns_suffix="azureiotcentral.com",
+    device_status=None,
 ):
-    provider = CentralDeviceProvider(cmd=cmd, app_id=app_id, token=token)
+    provider = CentralDeviceProvider(cmd=cmd, app_id=app_id, token=token,)
     if not device_id:
-        return provider.get_all_registration_info(central_dns_suffix=central_dns_suffix)
+        return provider.get_all_registration_info(
+            central_dns_suffix=central_dns_suffix, device_status=device_status
```

Comment:
What pattern are we following, new line for parameter or on the same line?

Reply:
We use an auto-formatter called "Black", that tool is deciding the spacing for us